### PR TITLE
20801 - Framework for updated Significant Individual form

### PR DESCRIPTION
**Pull request by hfekete**
*Issue:20801*https://github.com/bcgov/entity/issues/20801

*Description of changes:*
- reorg some code to support sections
- separated add SI into sections

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-145-00bh980h.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-145-00bh980h.web.app

File: btr-web/btr-main-app/cypress/e2e/pages/add-individual/external-influence.cy.ts [hunk 1]
```diff
@@ -1,42 +0,0 @@
-describe('pages -> Add individual', () => {
-  beforeEach(() => {
-    cy.visit('/')
-  })
-
-  it('renders the External Influence section and default radio is selected', () => {
```
Comment (kialj876) [inline btr-web/btr-main-app/cypress/e2e/pages/add-individual/external-influence.cy.ts:6]: again just verifying that the design wants to remove this from the form? If not we should keep these tests

**Review by kialj876 — COMMENTED** [review]
Hey nice work! I have some minor comments / questions. Also I think in each section with a subtitle (i.e. *Full Legal Name*) there is top padding that we don't want anymore. Removing that will put it inline with the section title

File: btr-web/btr-main-app/components/individual-person/add-new/index.vue [hunk 2]
```diff
@@ -186,79 +246,47 @@
...
-          {{ $t('texts.taxResidency') }}
-        </p>
-        <IndividualPersonTaxInfoTaxResidency
-          id="addNewPersonTaxResidency"
-          v-model="significantIndividual.profile.isTaxResident"
-          :errors="taxResidencyErrors"
-          data-cy="testTaxResidency"
-        />
-      </div>
-      <div data-cy="significant-individual-external-influence">
```
[the remark this answers is not in the document]
Comment (hfekete) [reply btr-web/btr-main-app/components/individual-person/add-new/index.vue:211]: Yeah, they decided that external influence is not a thing anymore. I think we will need to update the Summary table too.

[the remark this answers is not in the document]
Comment (hfekete) [reply btr-web/btr-main-app/components/individual-person/add-new/index.vue:211]: #20839 ticket for updating summary table

File: btr-web/btr-main-app/components/individual-person/add-new/section.vue [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+          {{ sectionTitle }}
+        </label>
+      </div>
+    </slot>
+    <slot name="default" />
+  </div>
+</template>
+
+<script setup lang="ts">
+import { defineProps } from 'vue'
```
[the remark this answers is not in the document]
Comment (hfekete) [reply btr-web/btr-main-app/components/individual-person/add-new/section.vue:18]: yeah, didnt remove it as my IDE automatically adds imports for me.
Will remove

**Review by kialj876 — APPROVED** [review]
Thanks for the updates!

File: btr-web/btr-main-app/components/individual-person/add-new/section.vue [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<template>
```
[the remark this answers is not in the document]
Comment (hfekete) [reply btr-web/btr-main-app/components/individual-person/add-new/section.vue]: moved it

**Pull request merged**